locale/de_DE/LC_MESSAGES/djangojs.po

```diff
@@ -1096,6 +1096,40 @@ msgstr ""
 "Wenn Sie diese Seite verlassen ohne zu speichern, gehen die von Ihnen "
 "vorgenommenen Änderungen verloren."
 
+#: meinberlin/react/account/FeedItem.jsx:8
+msgid "now"
+msgstr ""
+
+#: meinberlin/react/account/FeedItem.jsx:10
+msgid "%(minutes)sm ago"
+msgstr ""
+
+#: meinberlin/react/account/FeedItem.jsx:15
+msgid "%(hours)sh ago"
+msgstr ""
+
+#: meinberlin/react/account/FeedItem.jsx:19
+msgid "Yesterday"
+msgstr ""
```

Comment:
shouldn't those be translated too?

Reply:
Yes, but we need to merge this PR so PMs can translate it on weblate